src/launchpad/models/apple.py

```diff
@@ -37,6 +37,8 @@ class AppleAppInfo(BaseAppInfo):
         None, description="Type of codesigning used (development, adhoc, appstore, enterprise)"
     )
     profile_name: str | None = Field(None, description="Name of the provisioning profile used")
+    is_valid: bool = Field(True, description="Whether the app's code signature is valid")
+    errors: List[str] = Field(default_factory=list, description="List of code signature validation errors")
```

Comment:
Same with this -> `code_signature_errors`, or can alternatively place in a nested model.